src/common/datetime.cpp

```diff
@@ -541,6 +541,52 @@ bool wxDateTime::IsLeapYear(int year, wxDateTime::Calendar cal)
     }
 }
 
+#ifdef _WIN32
+#include <wx/msw/registry.h>
+
+/* static */
+wxDateTime::WeekDay wxDateTime::GetFirstWeekDay(bool *success) {
```

Comment:
Ideal would be to follow wx coding style, with braces on their own line (for functions, but also for `if` statements etc).